utils/resource-monitor/README.md

```diff
@@ -0,0 +1,36 @@
+# Resource performance monitoring and graphing module
+
+This is a module for monitoring the resource usage of batch mode with HAPI as the source. The scripts in this module generate graphs and csv files of resource usage (CPU, memory and I/O) of the HAPI server, postgres database and pipeline over the duration of the batch job. The user is able to specify the number of processes/cores used in the pipeline to assess batch mode's performance on the local machine.
+
+## Prerequisites
+
+-   Ensure you are able to run the `docker` command as a non-root user. If not, run <br />
```

Comment:
Please add a link to docker documentation regarding this and its implications; maybe this: https://docs.docker.com/engine/install/linux-postinstall/